samples/maps/demo/flowmap-around-the-world/demo.html

```diff
@@ -0,0 +1,7 @@
+<script src="https://code.highcharts.com/maps/highmaps.js"></script>
+<script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
```

Comment:
Just a suggestion, maybe we should import _exporting modules_ as the last ones here? Might be a tiny bit better in terms of clarity.